ludwig/features/binary_feature.py

```diff
@@ -15,18 +15,18 @@
 # limitations under the License.
 # ==============================================================================
 import logging
-
 import numpy as np
 # import tensorflow as tf
```

Comment:
Could you remove the commented out code from this file? Also at lines 20, 30, 136, etc.

Reply:
removed all commented out code